### PR TITLE
Bugfix/fixes

### PR DESCRIPTION
**Pull request by greenblitz4590**

### CODE REVIEW

**Review by aaa-png — APPROVED** [review]

File: src/main/java/edu/greenblitz/robotName/OI.java
Comment (yonnbro) [inline src/main/java/edu/greenblitz/robotName/OI.java:1]: so many imports without use? if really unused - remove

File: src/main/java/edu/greenblitz/robotName/subsystems/arm/Elbow.java [hunk 1]
```diff
@@ -113,7 +115,7 @@ public boolean isInDangerZone() {
     public Pose3d getPose3D (){
         return new Pose3d(
                 ElbowConstants.ELBOW_POSITION_RELATIVE_TO_ROBOT,
-                new Rotation3d(elbowInputs.position,0, 0)
+                new Rotation3d(elbowInputs.position + SimulationElbowConstants.SIMULATION_NUDGE,0, 0)
```
Comment (yonnbro) [inline src/main/java/edu/greenblitz/robotName/subsystems/arm/Elbow.java:118]: move this NUDGE to init in simulation, and rename it to offset

File: src/main/java/edu/greenblitz/robotName/subsystems/arm/ElbowUtils/ElbowConstants.java [hunk 2]
```diff
@@ -22,11 +22,11 @@ public enum PresetPositions {
...
     public static final double ARM_MASS_KG = 1;
 
-    public static final Rotation2d BACKWARD_ANGLE_LIMIT = Rotation2d.fromDegrees(-180);
+    public static final Rotation2d BACKWARD_ANGLE_LIMIT = Rotation2d.fromDegrees(5);
 
-    public static final Rotation2d FORWARD_ANGLE_LIMIT = Rotation2d.fromDegrees(20);
+    public static final Rotation2d FORWARD_ANGLE_LIMIT = Rotation2d.fromDegrees(180);
 
-    public static final Pair<Rotation2d,Rotation2d> DANGER_ZONE = new Pair<>(Rotation2d.fromDegrees(0),FORWARD_ANGLE_LIMIT);
+    public static final Pair<Rotation2d,Rotation2d> DANGER_ZONE = new Pair<>(BACKWARD_ANGLE_LIMIT,Rotation2d.fromDegrees(20));
```
Comment (yonnbro) [inline src/main/java/edu/greenblitz/robotName/subsystems/arm/ElbowUtils/ElbowConstants.java:29]: explain what is DANGER_ZONE what why it is not between BACKWARD_ANGLE_LIMIT and FORWARD_ANGLE_LIMIT

File: src/main/java/edu/greenblitz/robotName/subsystems/arm/ElbowUtils/SimulationElbow/SimulationElbowConstants.java [hunk 3]
```diff
@@ -11,4 +12,6 @@ public class SimulationElbowConstants {
 
     public static final PIDObject SIMULATION_PID = new PIDObject().withKp(0.3).withKd(0).withMaxPower(1);
 
+    public static final double SIMULATION_NUDGE = - Units.degreesToRadians(90);
```
Comment (yonnbro) [inline src/main/java/edu/greenblitz/robotName/subsystems/arm/ElbowUtils/SimulationElbow/SimulationElbowConstants.java:15]: same as above

File: src/main/java/edu/greenblitz/robotName/subsystems/arm/Wrist.java [hunk 4]
```diff
@@ -23,108 +21,109 @@
...
-	}
+    public double getDynamicFeedForward(double velocity) {
+        return NeoWristConstants.WRIST_FEED_FORWARD.calculate(velocity);
+    }
 
-	public double getVoltage() {
-		return wristInputs.appliedOutput * Battery.getInstance().getCurrentVoltage();
-	}
+    public double getVoltage() {
+        return wristInputs.appliedOutput * Battery.getInstance().getCurrentVoltage();
```
Comment (yonnbro) [inline src/main/java/edu/greenblitz/robotName/subsystems/arm/Wrist.java:98]: R we sure about the multi with batteryVoltage?

File: src/main/java/edu/greenblitz/robotName/subsystems/arm/Wrist.java [hunk 5]
```diff
@@ -23,108 +21,109 @@
...
-	}
+    public boolean isObjectInside() {
+        return wristInputs.isObjectInArm;
+    }
 
-	public boolean isAtAngle(Rotation2d angle) {
-		return Math.abs(angle.getRadians() - getAngleInRadians()) <= TOLERANCE;
-	}
+    public boolean isAtAngle(Rotation2d angle) {
+        return Math.abs(angle.getRadians() - getAngleInRadians()) <= TOLERANCE;
```
Comment (yonnbro) [inline src/main/java/edu/greenblitz/robotName/subsystems/arm/Wrist.java:114]: we should "minimize" the input angle (modulo 360)

File: src/main/java/edu/greenblitz/robotName/subsystems/arm/Wrist.java [hunk 6]
```diff
@@ -23,108 +21,109 @@
...
+    }
 
     public Pose3d getPose3D() {
         Translation3d elbowTranslation = Elbow.getInstance().getPose3D().getTranslation();
-        double trueElbowAngle = -Elbow.getInstance().getAngleInRadians() - Math.PI / 2;
+        double trueElbowAngle = -(Elbow.getInstance().getAngleInRadians() + Math.PI / 2 + SimulationElbowConstants.SIMULATION_NUDGE);
```
Comment (yonnbro) [inline src/main/java/edu/greenblitz/robotName/subsystems/arm/Wrist.java:119]: same SIMULATION_NUDGE comment

Comment (yonnbro) [inline src/main/java/edu/greenblitz/robotName/subsystems/arm/Wrist.java:119]: have a consts for the minus sign so everyone can understand why it is there

**Review by yonnbro — DISMISSED** [review]

**Review by tnrz135 — APPROVED** [review]

**Pull request merged**